### PR TITLE
Metadata Repair

### PR DESCRIPTION
**Pull request by MichaelJ2324**
Resolves missing Change Log buttons in modules that have Change Log
enabled. Need to add tests, but I tested it in an affected instance and
it works.

### CODE REVIEW

File: src/SugarModules/modules/supp_SugarRepairs/Classes/Repairs/supp_ReportRepairs.php [hunk 1]
```diff
@@ -16,6 +16,29 @@ function __construct()
     }
 
     /**
+     * Prefixes a given report with Broken: to let users know problematic reports
+     * @param $id
+     */
+    public function markReportBroken($id)
```
Comment (geraldclark) [inline src/SugarModules/modules/supp_SugarRepairs/Classes/Repairs/supp_ReportRepairs.php]: please move these function back to the main repair class. although they are obviously related to certain functionality reports, other repairs may need to use this in their logic.

File: src/SugarModules/modules/supp_SugarRepairs/Classes/Repairs/supp_Repairs.php [hunk 2]
```diff
@@ -631,29 +631,6 @@ public function setTesting($isTesting)
     }
 
     /**
-     * Prefixes a given report with Broken: to let users know problematic reports
-     * @param $id
-     */
-    public function markReportBroken($id)
```
Comment (geraldclark) [inline src/SugarModules/modules/supp_SugarRepairs/Classes/Repairs/supp_Repairs.php]: please move back

File: src/SugarModules/modules/supp_SugarRepairs/Classes/Repairs/supp_Repairs.php [hunk 3]
```diff
@@ -670,53 +647,6 @@ public function updateQuery($sql)
     }
 
     /**
-     * Disables a specific workflow
-     * @param $id
-     */
-    public function disableWorkflow($id)
```
Comment (geraldclark) [inline src/SugarModules/modules/supp_SugarRepairs/Classes/Repairs/supp_Repairs.php]: please move back

File: src/SugarModules/modules/supp_SugarRepairs/Classes/Repairs/supp_Repairs.php [hunk 4]
```diff
@@ -995,56 +925,6 @@ public function getValidLanguageKeyName($key)
     }
 
     /**
-     * Gets all time period IDs not deleted
-     * @return array $timePeriodIds
-     */
-    public function getAllTimePeriodIds()
```
Comment (geraldclark) [inline src/SugarModules/modules/supp_SugarRepairs/Classes/Repairs/supp_Repairs.php]: please move back

File: src/SugarModules/modules/supp_SugarRepairs/Classes/Repairs/supp_ReportRepairs.php (hunk 1 above)
Comment (MichaelJ2324) [inline src/SugarModules/modules/supp_SugarRepairs/Classes/Repairs/supp_ReportRepairs.php]: It'd make more sense to keep them in their respective class and set them as static, which was why I made this change at this point. Once they are static however, we would need to rework the Logging portion of this however, which would be useful anyway. 

I'm reverting them for now.

**Pull request merged**